src/client/components/menu/Menu.component.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { BrowserRouter, Link } from 'react-router-dom';
+import './Menu.styles.css';
+import faUser from '../../assets/images/user-login.png';
+import faHeart from '../../assets/images/favorite-icon.png';
+import faShoppingCart from '../../assets/images/shopping-cart.png';
+
```

Comment:
It is going in a good direction, but we are currently we are missing the logic for knowing if the user is authenticated or not. You need to add props, fx.  isAuthenticated, and then you can organize you code to fit the two scenarios: the user is authenticated or the user is not authenticated.

Reply:
Looks good now.